Makefile

```diff
@@ -48,6 +48,10 @@ test: node_modules bindings/node/test.js
 	$(NODE) ./$</ajv-cli/dist/index.js test --valid --spec=draft2020 -s schemas/rule-test.json -d test/rules/jsonschema-2019-09-to-2020-12.json
 	$(NODE) ./$</tap/bin/run.js --jobs=1 --reporter=list --no-coverage --no-timeout $(word 2,$^)
 
+
+serve:
+	webpack serve --open
```

Comment:
Can we do `$(NODE) ...` and the path to webpack instead to make it clear and make sure we spawn it using the node version defined in this Makefile?